llvm/test/DebugInfo/Generic/multi-variant.ll

```diff
@@ -0,0 +1,74 @@
+; RUN: %llc_dwarf -O0 -filetype=obj < %s > %t
+; RUN: llvm-dwarfdump -v -debug-info %t | FileCheck %s
+
+; Check for a variant part where a variant has multiple members.
+
+; CHECK: DW_AT_name [DW_FORM_str{{[a-z]+}}]  ({{.*}} = "Discr")
```

Comment:
@tromey @dwblaikie, please take care to check the regex pattern in future patches.
I have committed a fix here: https://github.com/llvm/llvm-project/commit/64e36da23248b6817e036b5d2c9fa5afb9f2c5e5

Reply:
Thanks @hubert-reinterpretcast - did this show up in any buildbot failures/elsewhere (or did you discover it locally)? I couldn't immediately find it in the two failures linked above which looked like sanitizer test flakes - but I do struggle to search through those reports to find the meaningful parts sometimes, so I may've missed it and would like to learn more about where I should've looked, if it is in there.

Reply:
FWIW it showed up on AIX for my previous patch, and then when I copy-pasted the test case for the new patch, I copied from the wrong version.  The issue for AIX was that it used a different form for the string constants.

Reply:
> did this show up in any buildbot failures/elsewhere (or did you discover it locally)?

From the failing buildbot, the log for one of the failing runs is: https://lab.llvm.org/buildbot/#/builders/64/builds/3545/steps/6/logs/FAIL__LLVM__multi-variant_ll